R/rdeephaven/src/client.cpp

```diff
@@ -157,7 +157,7 @@ class ClientWrapper {
 
         auto wrapper = internal_tbl_hdl_mngr.createFlightWrapper();
         arrow::flight::FlightCallOptions options;
-        wrapper.addAuthHeaders(&options);
+        wrapper.addHeaders(&options);
```

Comment:
Why this change? Are we positive this is not a breaking change?

Reply:
This is not arrow's API, is our DH C++ client's API.  The version on head already changed, the `addAuthHeaders` method doesn't exist anymore, it was renamed `addHeaders` (because it does that, it adds headers, they don't need to be related to auth).